storage/tianmu/handler/ha_rcengine.cpp

```diff
@@ -144,5 +144,47 @@ bool tianmu_load(THD *thd, sql_exchange *ex, TABLE_LIST *table_list, void *arg)
   return false;
 }
 
+bool TianmuDeleteMultiTbl(THD *thd, TABLE_LIST *table_list, ha_rows &deleted){
+
+  if (!core::Engine::IsTianmuTbl(table_list->table)) return true;
```

Comment:
Adding fault injection codes, just like MySQL does. What will happens that if the instance crash in deleting operation?

Reply:
And how about deleting rows from a temperoary table?